src/main/java/core/basesyntax/FigureColor.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class FigureColor implements Figure {
```

Comment:
The abstract class 'FigureColor' should not implement the 'Figure' interface if it is only meant to provide color functionality. The 'Figure' interface should be implemented by concrete figure classes.